extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.js

```diff
@@ -232,7 +261,7 @@ export class AmpLightboxViewer extends AMP.BaseElement {
    * @private
    */
```

Comment:
Add the `@return` annotation now that the function returns a promise.

Reply:
I think this doesn't apply to this function in particular, but I've gone through this file and added `@return` types. This function doesn't need to return anything.